projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.ts

```diff
@@ -11,6 +11,7 @@ import {
   GlobalMessageType,
   TranslationService,
 } from '@spartacus/core';
+import { getAddressNumbers } from 'feature-libs/checkout/base/core/utils/utils';
```

Comment:
This import statement is wrong - absolute paths should be in this format: `import * from '@spartacus/checkout/base/core';`.
However, this should not import from a feature module.

You will need to "getAddressNumbers" to somewhere else... perhaps under "projects/storefrontlib/utils". Any thoughts @Zeyber ?